documentation/staking-governance.md

```diff
@@ -13,21 +13,21 @@ This design with selectable locking duration helps to align token holders with t
+- Users can **modify** their locking positions (increase amount, extend locking duration) at any time.
+- Users have the option to **pause** their locking period countdown. That means, the remaining locking duration remains fixed until the user decides to resume it. Note that this results in higher voting power and staking rewards as described below. 
 - Each user may have multiple locking positions. 
-- Locking positions are represented using NFTs for composability into DeFi and potential future uses. 
-- In case a user wants to unlock earlier than the end of the locking duration, there is a fast unlock option. The fast unlock implies a penalty, i.e., an amount that is deducted from the locked LSK tokens. The penalty is set to `0.5 * lockedAmount *  (remainingLockingDurationInDays / maximumLockingDuration)`, where `maximumLockingDuration` is set to 2 years. Users can then redeem their tokens after a 3-day emergency locking period.
+- Locking positions are represented using **NFTs** for composability into DeFi and potential future uses. 
+- In case a user wants to unlock earlier than the end of the locking duration, there is a **fast unlock** option. The fast unlock implies a penalty, i.e., an amount that is deducted from the locked LSK tokens. The penalty is set to `0.5 * lockedAmount *  (remainingLockingDurationInDays / maximumLockingDuration)`, where `maximumLockingDuration` is set to 2 years. Users can then redeem their tokens after a 3-day emergency locking period.
   - Confiscated tokens are immediately redirected to the staking rewards pool to be distributed over the next 14 days period on top of the guaranteed rewards.
   - Users do not have the option to cancel the fast un-stake during this 3 day window.
 
 ## Voting Power
 
-The voting power for a locked amount of tokens is computed as follows:
+The voting power of an account is expressed by the account's balance of another ERC20 contract, the *Voting Power* contract. The token symbol of this contract `vpLSK`. The voting power for a locked amount of tokens is computed as follows:
 
-- Generally, locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides one unit of voting power.
-- In case a user pauses their locking period countdown, they receives a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200%.
+- Generally, locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides 1 vpLSK.
+- In case a user pauses their locking period countdown, they receive a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200% by pausing a locking position with the maximum locking period of 2 years.
```

Comment:
Minor suggestion for improvement: 
```suggestion
- In case a user pauses their locking period countdown, they receive a boost of the voting power. In particular,  their voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)` vpLSK. Hence, the voting power can be increased by up to 200% by pausing a locking position with the maximum locking period of 2 years.
```

Reply:
I was also thinking about adding the unit `vpLSK` here. But as `lockedAmount` could also be understood as the variable used on the contract level, which is in wei, I decided against it.